Assets/Scripts/Control&Input/PlayerInputManagerController.cs

```diff
@@ -107,6 +109,9 @@ public void ChangeInputMaps(string mapNameOrId)
     {
         LocalPlayerInputs.ForEach(playerInput =>
         {
+            Cursor.visible = playerInput.IsMouseAndKeyboard && mapNameOrId.Equals("Menu");
```

Comment:
I'm not sure why this works? Shouldn't this cause the cursor to be invisible if the last input is a gamepad?

I'd suggest writing it as
```csharp
Cursor.visible = LocalPlayerInputs.Any(i => i.IsMouseAndKeyboard) && mapNameOrId.Equals("Menu");
```
before the ForEach.